### DISCUSSION TITLE
Duplicate correction

### DISCUSSION THREAD

**Discussion by teknowledgist** [tom5760/ezproxy-db#13] [General]
There are two "University at Albany" entries:

* SUNY University at Albany
* University at Albany, State University of New York (SUNY)

The former has an incorrect URL.  If the URL is a key (used only once), then the former entry should be deleted and the latter item should remain.  If it is OK to have two names for the same URL, then the former should be named "SUNY Albany" to improve discovery (matches the "classic" name and parallels many other SUNYs) and the URL should equal that of the latter item.

Thanks!

**Comment by tom5760** [comment]
Hi!  While technically fine for two entries to have the same URL, it might be tidier to keep one, so we don't run into this issue again.  I'll update the entry to be the following:

```json
{
    "name": "SUNY Albany",
    "url": "https://libproxy.albany.edu/login?url=$@"
}
```

Does that look good?

**Reply by teknowledgist** [reply]
Would that be the only entry or just a cleanup of the first (of two) entry?  If you want to have only a single entry (which I totally understand), it should be:

````
{
    "name": "University at Albany, State University of New York (SUNY)",
    "url": "https://libproxy.albany.edu/login?url=$@"
}
````

As that is the (current) official name.  If two entries are allowed, what you suggested should be the additional one.

**Reply by teknowledgist** [reply]
Having just one is fine for users of the Chrome extension because it has search, but until that is added to the [FireFox extension](https://addons.mozilla.org/en-US/firefox/addon/ezproxy-redirect-foxified/) ([which I've requested](https://github.com/metronidazole/EZProxyRedirectFoxified/issues/1)), personally, I think the "SUNY Albany" is easier for folks to discover in the big list. 

_BTW, the link in the `readme.md` to the FireFox extension is broken._

**Reply by tom5760** [reply]
Unfortunately, someone else is running that Firefox extension.  I've been meaning to get in touch with them to re-sync with my version (or just use it directly, it supports Firefox now).

For now, I think I'd prefer to just have that one entry.  I'll update the database, and ask about updating the Firefox extension.

**Reply by tom5760** [reply]
Oh also, sorry I misread your original request.  I've updated the first entry to have the correct URL, and will leave the second alone for now at least.

I've also opened up a ticket [on the Firefox extension](https://github.com/metronidazole/EZProxyRedirectFoxified/issues/2) asking if they want to re-sync with the code over here.

**Reply by tom5760** [reply]
Thanks again for the note!
